docs/extend/install-configure-zos-extensions.md

```diff
@@ -148,3 +148,47 @@ EXTERNAL_COMPONENTS=some-other-extensions,myapp
 You might need to manually run the script `<INSTANCE_DIR>/bin/install-app.sh` if your component is a Desktop plug-in. Or you can choose to add this step to [Zowe component Configure lifecycle stage](lifecycling-with-zwesvstc.md#configure).
 
 When the Zowe instance is launched by running `<INSTANCE_DIR>/bin/zowe-start.sh`, it will read manifest `commands` instructions and call the `/usr/lpp/myvendor/myapp/zowe-ext/bin/start.sh` script. The started task will create an address space under `ZWESVSTC` for the vendor component.  When the Zowe instance is stopped, the address space is terminated.
+
+
+## Verify with `zowe-verify-component.sh` (Technical Preview)
+
+<Badge text="Technical Preview"/>
+
+_Note: This section is for technical preview and we are happy to hear any feedback. Content in this section may be changed or improved in the future._
+
+_Note: This feature is added with Zowe v1.22.0 release._
+
+From Zowe v1.22.0, Zowe ships a `bin/zowe-verify-component.sh` tool to help you configure an installed Zowe server component (extension) for a Zowe instance. Zowe core components are also configured with this tool. In order to be compatible with the tool, we recommend components follow [Zowe server component package format standard](packaging-zos-extensions.md#zowe-server-component-package-format).
```

Comment:
`Zowe core components are also configured with this tool.` is not applicable to this tool.

Reply:
Add ` and define [Zowe component manifest](packaging-zos-extensions.md#zowe-component-manifest).` following `we recommend components follow` statement.

Reply:
Instead of `help you configure an installed Zowe server component`, should it be `help you verify an installed Zowe server component`?